app/javascript/mastodon/reducers/conversations.js

```diff
@@ -74,6 +76,10 @@ const expandNormalizedConversations = (state, conversations, next, isLoadingRece
   });
 };
 
+const filterConversations = (state, accountIds) => {
+  return state.update('items', list => list.filterNot(item => item.get('accounts').some(accountId => accountIds.includes(accountId))));
+};
```

Comment:
Hm, I'm not still 100% sure about the “hide everything mentioning blocked users” logic (although it's not new).
People mentioning both you and someone you blocked might not know you've blocked that last person.
Also, not having notifications in those cases isn't new, but I think not being able to see the message at all is new? I'd have to check.